apiserver/userservice/userimpl.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/crunchydata/postgres-operator/util"
 
 	log "github.com/sirupsen/logrus"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I don't think we need the `v1` alias here as we get it for free!

Reply:
Leaving as is since this `goimports` is working as designed (https://github.com/golang/go/issues/30051).